### DISCUSSION TITLE
requests from Harvard / UW GMM

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/uwlswd_ngl#23] [requested_updates]
# request details
## Per UW GMM (JA) 2024-04-11, highest priority requests originating from Harvard are...

- **African American newspapers** –> **Black newspapers**
- **Handicapped persons’ newspapers** –> **Persons’ with disabilities newspapers**

...and there are additional requests for consideration:

> I would love it if you used one or two of the suggestions from the Harvard librarian that I feel confident making. They are below in bold.

> Black newspapers has become the standard term for librarians, journalists, and communication studies faculty. I consulted with Matt Parsons, our Disability Studies librarian and he agrees that Persons’ with disabilities newspapers should now be used.

> Native American newspapers –> American Indian newspapers
> **African American newspapers** –> **Black newspapers**
> **Handicapped persons’ newspapers** –> **Persons’ with disabilities newspapers**
> Alaska Native newspapers –> Eskimo newspapers

## Other requests from GMM staff:

- The preferred term “Native American newspapers” is missing:
   - NT Blackfoot
   - NT Cheyenne
   - NT Lammi
   - NT Yakama
- “Penny newspapers” is missing entirely
- “School newspapers” is missing:
   - RT Student newspapers
- Add “Chicano newspapers” as a NT under the “Hispanic Newspapers” genre and create a full preferred term (155) for “Chicano newspapers”

**Comment by AdamSchiff** [comment]
Are these changes now official?  If so, I'll need to revise OCLC records that used the former terms, and update Wikidata as well probably.

**Reply by briesenberg07** [reply]
Hi @AdamSchiff 
These changes are *not official*.  
I'm attempting to document changes that were requested previously but not made, so we know where things stand and what the "to-do list" is.

**Comment by briesenberg07** [comment]
# background

## 2023-03 from @E-Fl0  
> These suggestions were made by a librarian at Harvard in an effort to use more modern DEI terminology. I'd be curious what your thoughts are on these changes. This is probably something that needs to be added to whatever workflow is created around this thesaurus."

## 2021 email exchange between UW/Harvard

## Sent: Friday, May 28, 2021 10:40 AM
From: harvard.edu
Subject: RE: question about newspaper genre list
Thanks so much for the follow up. I’ll be in touch if I see any terms that I think have DEI concerns in the list.
Thanks again for maintaining such a wonderful list for the community.

## Sent: Friday, May 28, 2021 4:04 AM
From: uw.edu
To: harvard.edu
Subject: RE: question about newspaper genre list
It took longer than expected, but I finally updated the list.  Our updates did not focus on EDI, so I asked our principal cataloger if he would next help me to consider updating terms like “Handicapped persons’ newspapers” and if Black newspapers should now be used instead of African American newspapers. 
I asked him about those two terms.  Please feel free to email additional suggestions to me and I’d be happy to review them with him.

## Sent: Wednesday, May 19, 2021 10:54 AM
From: harvard.edu
To: uw.edu
Subject: RE: question about newspaper genre list
Thanks so much for the quick response. I’m so glad you all are on top of this. Thanks again for sharing your wonderful list with us, and I look forward to hearing from you in the future.

## Sent: Wednesday, May 19, 2021 1:15 PM
From: uw.edu
To: harvard.edu
Subject: RE: question about newspaper genre list
This is a very brief email to say that you have perfect timing!  I recently worked with our catalogers to revise the list.  I’m working with the collection in the library today, but plan to update the list with their suggestions tomorrow.  I’ll also read your email more carefully then and then email you back.

## Sent: Wednesday, May 19, 2021 9:38 AM
From: harvard.edu
To: uw.edu
Subject: question about newspaper genre list
I got your email from the website for the Newspaper Genre List (https://www.lib.washington.edu/gmm/collections/mcnews/ngl).
We, at Harvard, recently discovered this list and we think it’s a great resource. Since many typical newspapers don’t have a specific subject, this list provides patron access to the nature of the newspaper. So many thanks to you and UW for compiling and maintaining this list!
We also noticed that you have included some DEI aspects to the thesaurus. For example, you use the term “Alaska Native newspapers” instead of “Eskimo newspapers.” You also use “Native American newspapers” instead of “American Indian newspapers.”
We were wondering if you have a regular review of your terms, or a mechanism by which we could report terms if we thought they should be updated. For example, we see that you use the term “Handicapped persons' newspapers.” According to the NDA, the appropriate term would be “persons with disabilities” (http://nda.ie/Publications/Attitudes/Appropriate-Terms-to-Use-about-Disability/)
At Harvard we have been focusing on DEI issues, as have most universities across the country. So we noticed this term in your thesaurus, and felt it best to report it to you. If this request should go to someone else, please let me know.
